authenticator/server/credmgrs/util.go

```diff
@@ -0,0 +1,159 @@
+package credmgrs
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+)
+
+const (
+	userDataMapIamArnsKey = "iam_arns"
+	userDataMapPwKey      = "password"
+)
+
+type DBKey struct {
```

Comment:
TODO replace this with `PasswordRequest` proto, and use it for getting a `Password` through that ifc method.